.mergify.yml

```diff
@@ -4,7 +4,6 @@ defaults:
     comment:
       bot_account: ceph-csi-bot
     merge:
-      bot_account: ceph-csi-bot
```

Comment:
The `merge` action has options for `merge_bot_account` and `update_bot_account` now. These are part of the premium support plan. I would prefer to set those, even if we  do not have such support (or do we have that through Ceph?) and they might not be used (add a comment about it).

Could you test that out, and set them?

See https://docs.mergify.io/actions/merge/

Reply:
`merge_bot_account` is used to set which account(user) to be used to merge pr. It was set in another pr and had to be reverted(we do need subscription for it). 
https://github.com/ceph/ceph-csi/pull/1981#issue-615990483

https://docs.mergify.io/actions/merge/ 
`merge_bot_account` :Mergify can impersonate a GitHub user to merge pull request. **If no merge_bot_account is set, Mergify will merge the pull request itself.** The user account must have already been logged in Mergify dashboard once and have write or maintain permission.
`update_bot_account` :For certain actions, such as rebasing branches, Mergify has to impersonate a GitHub user. You can specify the account to use with this option. **If no update_bot_account is set, Mergify picks randomly one of the organization users instead.** The user account must have already been logged in Mergify dashboard once.

In the absence of both the options, Mergifyio will have the same behavior as before.

Reply:
Yes, and **Mergify picks randomly one of the organization users instead.** is what causes confusion sometimes. It is fine if **mergify** does the merges and updates, but a random user from the organization (for us ceph, not ceph/ceph-csi) is a little annoying as some users that do not contribute to ceph-csi (but other ceph projects) suddenly get listed/notified here.